filament/backend/src/opengl/platforms/PlatformEGL.cpp

```diff
@@ -86,13 +86,28 @@ Driver* PlatformEGL::createDriver(void* sharedContext) noexcept {
 
     EGLint major, minor;
     EGLBoolean initialized = eglInitialize(mEGLDisplay, &major, &minor);
+
+    if(!initialized) {
+      EGLDeviceEXT eglDevice;
+      EGLint numDevices;
+
+      PFNEGLQUERYDEVICESEXTPROC eglQueryDevicesEXT =
+              (PFNEGLQUERYDEVICESEXTPROC)eglGetProcAddress("eglQueryDevicesEXT");
+      if (eglQueryDevicesEXT != NULL) {
+          eglQueryDevicesEXT(1, &eglDevice, &numDevices);
+          mEGLDisplay = eglGetPlatformDisplay(EGL_PLATFORM_DEVICE_EXT, eglDevice, 0);
```

Comment:
You can't use `eglGetPlatformDisplay` because it doesn't exist in all versions of EGL -- I think you need to protect it with some #if on the EGL version.